app/models/ability.rb

```diff
@@ -64,6 +65,7 @@ def define_school_teacher_abilities(user:, school:)
     can(%i[read], School, id: school.id)
     can(%i[create], SchoolClass, school: { id: school.id })
     can(%i[read update destroy], SchoolClass, school: { id: school.id }, teacher_id: user.id)
+    can(%i[read], Project, school_id: school.id, lesson: { visibility: %w[teachers students public] })
```

Comment:
It looks like this also includes the "public" visibility but I can't see that path tested in the `ability_spec`, would you be able to add one?

Reply:
I'd be tempted on removing the `public` visibility for now, I don't think it'll be used and will become apparent if it does

Reply:
👍